src/store/domains/ui.ts

```diff
@@ -139,7 +150,7 @@ export default class UI implements UIStoreInterface, RoutingObservableInterface
   }
 
   setArtifactKind(artifactKind: UIArtifactKind): void {
-    if (this.artifactKind === artifactKind) return;
+    this.setSideBarContent(UISidebarContentType.FILTER);
```

Comment:
Hiermit erzwingen wir immer den Sprung zur Suche/Filterung in der Sidebar, wenn der Artefakttyp geändert wird. Sei es durch den User über das Menü oben links.